scheduler/bin/gcloud-make-for-integration.sh

```diff
@@ -0,0 +1,56 @@
+##
+## Configure a kubernetes cluster for running pool-based integration tests and running pools in general.
+##
+##
+
+
+set -e
+
+## Example arguments that work:
+##
+
+#ZONE=us-central1-a
+#CLUSTERNAME=test-cluster-2
+#PROJECT=<FILL SOMETHING IN>
+
+PROJECT=$1
+ZONE=$2
+CLUSTERNAME=$3
+
+echo "---- Building kubernetes cluster for project=$PROJECT zone=$ZONE clustername=$CLUSTERNAME"
+
+# Nuke all existing temporary clusters; don't want to keep on making more idle clusters each time you invoke this.
+echo "---- Deleting any existing temporary clusters."
+echo "---- Will delete the following. If you don't want them gone, control-c now"
+gcloud container clusters list --filter 'resourceLabels.longevity=temporary'
+sleep 10
```

Comment:
This is supposed to be automated. I don't like deleting without a (small) warning. But I also don't want this to pause forever for interactive input.

Reply:
How about ask for confirmation but add `--force` to skip confirmation?

Reply:
@scrosby Thoughts on this?

Reply:
On discussion, adding --force and a prompt isn't worth the development effort. We'll remove the sleep instead and it will immediately delete on invocation.